docs/code/MixingITKAndNumPy.py

```diff
@@ -14,34 +14,40 @@
 # Run filters on itk.Image
 
 # View only of itk.Image, data is not copied
-np_view = itk.GetArrayViewFromImage(itk_image)
+np_view = itk.array_view_from_image(itk_image)
 
 # Copy of itk.Image, data is copied
-np_copy = itk.GetArrayFromImage(itk_image)
+np_copy = itk.array_from_image(itk_image)
 
 
-# Do numpy stuff
+# Do numpy stuff...
 
 
 # Convert back to itk, view only, data is not copied
```

Comment:
`itk` has been capitalized in other comments. I would use the capitalized version everywhere.

Reply:
Good point -- capitialized!